modules/op-disabling-tekton-hub-authorization-after-upgrade.adoc

```diff
@@ -0,0 +1,81 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="disabling-tekton-hub-authorization-after-upgrade.adoc_{context}"]
+= Disabling {tekton-hub} authorization after upgrading the {pipelines-title} Operator from 1.7 to 1.8
+
+[role="_abstract"]
+When you install {tekton-hub} with {pipelines-title} Operator 1.8, the login authorization and ratings for the Hub artifacts are disabled for the default installation. However, when you upgrade the Operator from 1.7 to 1.8, the instance of the Hub on your cluster does not automatically disable the login authorization and ratings. 
+
+To disable login authorization and ratings for {tekton-hub} after upgrading the Operator from 1.7 to 1.8, perform the steps in the following procedure.
+
+[discrete]
+.Prerequisites
+* Ensure that the {pipelines-title} Operator is installed in the default `openshift-pipelines` namespace on the cluster.
+
+[discrete]
+.Procedure 
+
+Starting with {pipelines-title} 1.8, you can automatically install and deploy {tekton-hub} with default configurations on a cluster by performing the following steps:
+
+. Delete the existing {tekton-hub} API secret that you created while manually installing {tekton-hub} for Operator 1.7.
++
+[source,terminal]
+----
+$ oc delete secret tekton-hub-api -n <targetNamespace> <1>
+----
+<1> The common namespace for the {tekton-hub} API secret and the {tekton-hub} CR. By default, the target namespace is `openshift-pipelines`.
+
+. Delete the `tektoninstallerset` object for the {tekton-hub} API.
```

Comment:
When referring to the object, I think it should be ticked camel case - the `TektonInstallerSet` object - even though the command line itself uses lower case